### DISCUSSION TITLE
Streaming webcam feed into HoloLens

### DISCUSSION THREAD

**Discussion by destarkable** [StereoKit/StereoKit#509] [Q&A]
Is there a function to stream a camera feed into HoloLens? Since the Video Player component in Unity only provides local file and online resource playback.

**Comment by maluoi** [comment]
For HoloLens, my recommendation is to use the UWP [MediaCapture API](https://learn.microsoft.com/en-us/uwp/api/windows.media.capture.mediacapture?view=winrt-22621)! It can be a bit tough to navigate, but it does give you a lot of control over capturing videos and photos with the hardware. You can find a bit of information related to MediaCapture [over here](https://github.com/StereoKit/StereoKit/issues/168), though that thread is mostly focused on the photo use-case.

StereoKit does not have any built-in functions for working with webcams! I expect a cross-platform solution to that is a pretty big project, and possibly even a product of its own. You may also have some luck peeking around the NuGet market for other solutions!
